spacelift/provider.go

```diff
@@ -50,23 +50,27 @@ func Provider(commit, version string) plugin.ProviderFunc {
 				},
 			},
 			DataSourcesMap: map[string]*schema.Resource{
-				"spacelift_aws_role":                  dataAWSRole(),
-				"spacelift_context_attachment":        dataContextAttachment(),
-				"spacelift_context":                   dataContext(),
-				"spacelift_current_stack":             dataCurrentStack(),
-				"spacelift_drift_detection":           dataDriftDetection(),
-				"spacelift_environment_variable":      dataEnvironmentVariable(),
-				"spacelift_gcp_service_account":       dataGCPServiceAccount(),
-				"spacelift_ips":                       dataIPs(),
-				"spacelift_module":                    dataModule(),
-				"spacelift_mounted_file":              dataMountedFile(),
-				"spacelift_policy":                    dataPolicy(),
-				"spacelift_stack":                     dataStack(),
-				"spacelift_webhook":                   dataWebhook(),
-				"spacelift_stack_aws_role":            dataStackAWSRole(),           // deprecated
-				"spacelift_stack_gcp_service_account": dataStackGCPServiceAccount(), // deprecated
-				"spacelift_worker_pool":               dataWorkerPool(),
-				"spacelift_worker_pools":              dataWorkerPools(),
+				"spacelift_aws_role":                         dataAWSRole(),
+				"spacelift_context_attachment":               dataContextAttachment(),
+				"spacelift_context":                          dataContext(),
+				"spacelift_current_stack":                    dataCurrentStack(),
+				"spacelift_drift_detection":                  dataDriftDetection(),
+				"spacelift_environment_variable":             dataEnvironmentVariable(),
+				"spacelift_gcp_service_account":              dataGCPServiceAccount(),
+				"spacelift_ips":                              dataIPs(),
+				"spacelift_module":                           dataModule(),
+				"spacelift_mounted_file":                     dataMountedFile(),
+				"spacelift_policy":                           dataPolicy(),
+				"spacelift_stack":                            dataStack(),
+				"spacelift_webhook":                          dataWebhook(),
+				"spacelift_stack_aws_role":                   dataStackAWSRole(),           // deprecated
+				"spacelift_stack_gcp_service_account":        dataStackGCPServiceAccount(), // deprecated
+				"spacelift_worker_pool":                      dataWorkerPool(),
+				"spacelift_worker_pools":                     dataWorkerPools(),
+				"spacelift_gitlab_integration":               dataGitlabIntegration(),
```

Comment:
nit: we typically sort stuff like this alphabetically rather than just appending to the end.